Code/Tools/ProjectManager/Source/DownloadRemoteTemplateDialog.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <ProjectTemplateInfo.h>
+
+#include <QDialog>
+#endif
+
+QT_FORWARD_DECLARE_CLASS(QLabel)
+QT_FORWARD_DECLARE_CLASS(QCheckBox)
+QT_FORWARD_DECLARE_CLASS(QDialogButtonBox)
+QT_FORWARD_DECLARE_CLASS(QPushButton)
+QT_FORWARD_DECLARE_CLASS(QTimer)
```

Comment:
is this really necessary. seems just as good to me to do `class QLabel`.

Reply:
This was to be consistent with the rest of Project Manager, I'll add a GHI to cover removing all of these since there's no reason not to just use class <classname>.
However this was also forward declaring more classes that was needed, removed the additional ones.